scripts/ModalComponent.js

```diff
@@ -0,0 +1,17 @@
+import Button from './ButtonComponent';
+
+export default function(modal, container) {
+  function openModal() {
+    modal.classList.add("show");
+    container.classList.add("blur");
+  }
+  
+  function closeModal() {
+    modal.classList.remove("show");
+    container.classList.remove("blur");
+  }
+  
+  Button(document.getElementById("open-modal"), openModal);
```

Comment:
Whenever a component references another component, it couples them tightly together and makes it hard to scale the app.

Even if you keep them in the same component, I think the button should emit an `open-modal` event on `pubSub` rather than calling this function, and instead you should call the `openModal` by subscribing to the `open-modal` event. 

That way you can open the modal from anywhere, and you can do other things when the `open-modal` button is clicked (like clearing the input, or triggering an animation).